<?xml version="1.0" encoding="utf-8"?>
<ui version="4.0">
  <class>ConfigureInput</class>
  <widget class="QWidget" name="ConfigureInput">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>370</width>
        <height>435</height>
      </rect>
    </property>
    <property name="windowTitle">
      <string>ConfigureInput</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
        <layout class="QGridLayout" name="gridLayout_7">
          <item row="0" column="0">
            <widget class="QGroupBox" name="faceButtons">
              <property name="title">
                <string>Face Buttons</string>
              </property>
              <property name="flat">
                <bool>false</bool>
              </property>
              <property name="checkable">
                <bool>false</bool>
              </property>
              <layout class="QGridLayout" name="gridLayout">
                <item row="0" column="0">
                  <layout class="QVBoxLayout" name="verticalLayout">
                    <item>
                      <widget class="QLabel" name="label">
                        <property name="text">
                          <string>A:</string>
                        </property>
                      </widget>
                    </item>
                    <item>
                      <widget class="QPushButton" name="btnFaceA">
                        <property name="text">
                          <string />
                        </property>
                      </widget>
                    </item>
                  </layout>
                </item>
                <item row="0" column="1">
                  <layout class="QVBoxLayout" name="verticalLayout_2">
                    <item>
                      <widget class="QLabel" name="label_2">
                        <property name="text">
                          <string>B:</string>
                        </property>
                      </widget>
                    </item>
                    <item>
                      <widget class="QPushButton" name="btnFaceB">
                        <property name="text">
                          <string />
                        </property>
                      </widget>
                    </item>
                  </layout>
                </item>
                <item row="1" column="0">
                  <layout class="QVBoxLayout" name="verticalLayout_3">
                    <item>
                      <widget class="QLabel" name="label_3">
                        <property name="text">
                          <string>X:</string>
                        </property>
                      </widget>
                    </item>
                    <item>
                      <widget class="QPushButton" name="btnFaceX">
                        <property name="text">
                          <string />
                        </property>
                      </widget>
                    </item>
                  </layout>
                </item>
                <item row="1" column="1">
                  <layout class="QVBoxLayout" name="verticalLayout_4">
                    <item>
                      <widget class="QLabel" name="label_4">
                        <property name="text">
                          <string>Y:</string>
                        </property>
                      </widget>
                    </item>
                    <item>
                      <widget class="QPushButton" name="btnFaceY">
                        <property name="text">
                          <string />
                        </property>
                      </widget>
                    </item>
                  </layout>
                </item>
              </layout>
            </widget>
          </item>
          <item row="0" column="1">
            <widget class="QGroupBox" name="faceButtons_2">
              <property name="title">
                <string>Directional Pad</string>
              </property>
              <property name="flat">
                <bool>false</bool>
              </property>
              <property name="checkable">
                <bool>false</bool>
              </property>
              <layout class="QGridLayout" name="gridLayout_2">
                <item row="1" column="0">
                  <layout class="QVBoxLayout" name="verticalLayout_12">
                    <item>
                      <widget class="QLabel" name="label_34">
                        <property name="text">
                          <string>Up:</string>
                        </property>
                      </widget>
                    </item>
                    <item>
                      <widget class="QPushButton" name="btnDirUp">
                        <property name="text">
                          <string />
                        </property>
                      </widget>
                    </item>
                  </layout>
                </item>
                <item row="1" column="1">
                  <layout class="QVBoxLayout" name="verticalLayout_9">
                    <item>
                      <widget class="QLabel" name="label_35">
                        <property name="text">
                          <string>Down:</string>
                        </property>
                      </widget>
                    </item>
                    <item>
                      <widget class="QPushButton" name="btnDirDown">
                        <property name="text">
                          <string />
                        </property>
                      </widget>
                    </item>
                  </layout>
                </item>
                <item row="0" column="0">
                  <layout class="QVBoxLayout" name="verticalLayout_10">
                    <item>
                      <widget class="QLabel" name="label_32">
                        <property name="text">
                          <string>Left:</string>
                        </property>
                      </widget>
                    </item>
                    <item>
                      <widget class="QPushButton" name="btnDirLeft">
                        <property name="text">
                          <string />
                        </property>
                      </widget>
                    </item>
                  </layout>
                </item>
                <item row="0" column="1">
                  <layout class="QVBoxLayout" name="verticalLayout_11">
                    <item>
                      <widget class="QLabel" name="label_33">
                        <property name="text">
                          <string>Right:</string>
                        </property>
                      </widget>
                    </item>
                    <item>
                      <widget class="QPushButton" name="btnDirRight">
                        <property name="text">
                          <string />
                        </property>
                      </widget>
                    </item>
                  </layout>
                </item>
              </layout>
            </widget>
          </item>
          <item row="1" column="0">
            <widget class="QGroupBox" name="faceButtons_3">
              <property name="title">
                <string>Shoulder Buttons</string>
              </property>
              <property name="flat">
                <bool>false</bool>
              </property>
              <property name="checkable">
                <bool>false</bool>
              </property>
              <layout class="QGridLayout" name="gridLayout_3">
                <item row="0" column="0">
                  <layout class="QVBoxLayout" name="verticalLayout_13">
                    <item>
                      <widget class="QLabel" name="label_17">
                        <property name="text">
                          <string>L:</string>
                        </property>
                      </widget>
                    </item>
                    <item>
                      <widget class="QPushButton" name="btnShdrL">
                        <property name="text">
                          <string />
                        </property>
                      </widget>
                    </item>
                  </layout>
                </item>
                <item row="0" column="1">
                  <layout class="QVBoxLayout" name="verticalLayout_14">
                    <item>
                      <widget class="QLabel" name="label_19">
                        <property name="text">
                          <string>R:</string>
                        </property>
                      </widget>
                    </item>
                    <item>
                      <widget class="QPushButton" name="btnShdrR">
                        <property name="text">
                          <string />
                        </property>
                      </widget>
                    </item>
                  </layout>
                </item>
                <item row="1" column="0">
                  <layout class="QVBoxLayout" name="verticalLayout_15">
                    <item>
                      <widget class="QLabel" name="label_20">
                        <property name="text">
                          <string>ZL:</string>
                        </property>
                      </widget>
                    </item>
                    <item>
                      <widget class="QPushButton" name="btnShdrZL">
                        <property name="text">
                          <string />
                        </property>
                      </widget>
                    </item>
                  </layout>
                </item>
                <item row="1" column="1">
                  <layout class="QVBoxLayout" name="verticalLayout_16">
                    <item>
                      <widget class="QLabel" name="label_18">
                        <property name="text">
                          <string>ZR:</string>
                        </property>
                      </widget>
                    </item>
                    <item>
                      <widget class="QPushButton" name="btnShdrZR">
                        <property name="text">
                          <string />
                        </property>
                      </widget>
                    </item>
                  </layout>
                </item>
              </layout>
              <zorder></zorder>
            </widget>
          </item>
          <item row="1" column="1">
            <widget class="QGroupBox" name="faceButtons_4">
              <property name="title">
                <string>Circle Pad</string>
              </property>
              <property name="flat">
                <bool>false</bool>
              </property>
              <property name="checkable">
                <bool>false</bool>
              </property>
              <layout class="QGridLayout" name="gridLayout_4">
                <item row="0" column="0">
                  <layout class="QVBoxLayout" name="verticalLayout_17">
                    <item>
                      <widget class="QLabel" name="label_21">
                        <property name="text">
                          <string>Left:</string>
                        </property>
                      </widget>
                    </item>
                    <item>
                      <widget class="QPushButton" name="btnCircleLeft">
                        <property name="text">
                          <string />
                        </property>
                      </widget>
                    </item>
                  </layout>
                </item>
                <item row="0" column="1">
                  <layout class="QVBoxLayout" name="verticalLayout_18">
                    <item>
                      <widget class="QLabel" name="label_23">
                        <property name="text">
                          <string>Right:</string>
                        </property>
                      </widget>
                    </item>
                    <item>
                      <widget class="QPushButton" name="btnCircleRight">
                        <property name="text">
                          <string />
                        </property>
                      </widget>
                    </item>
                  </layout>
                </item>
                <item row="1" column="0">
                  <layout class="QVBoxLayout" name="verticalLayout_19">
                    <item>
                      <widget class="QLabel" name="label_24">
                        <property name="text">
                          <string>Up:</string>
                        </property>
                      </widget>
                    </item>
                    <item>
                      <widget class="QPushButton" name="btnCircleUp">
                        <property name="text">
                          <string />
                        </property>
                      </widget>
                    </item>
                  </layout>
                </item>
                <item row="1" column="1">
                  <layout class="QVBoxLayout" name="verticalLayout_20">
                    <item>
                      <widget class="QLabel" name="label_22">
                        <property name="text">
                          <string>Down:</string>
                        </property>
                      </widget>
                    </item>
                    <item>
                      <widget class="QPushButton" name="btnCircleDown">
                        <property name="text">
                          <string />
                        </property>
                      </widget>
                    </item>
                  </layout>
                </item>
              </layout>
            </widget>
          </item>
          <item row="2" column="0">
            <widget class="QGroupBox" name="faceButtons_5">
              <property name="title">
                <string>C-Stick</string>
              </property>
              <property name="flat">
                <bool>false</bool>
              </property>
              <property name="checkable">
                <bool>false</bool>
              </property>
              <layout class="QGridLayout" name="gridLayout_5">
                <item row="0" column="0">
                  <layout class="QVBoxLayout" name="verticalLayout_21">
                    <item>
                      <widget class="QLabel" name="label_25">
                        <property name="text">
                          <string>Left:</string>
                        </property>
                      </widget>
                    </item>
                    <item>
                      <widget class="QPushButton" name="btnStickLeft">
                        <property name="text">
                          <string />
                        </property>
                      </widget>
                    </item>
                  </layout>
                </item>
                <item row="0" column="1">
                  <layout class="QVBoxLayout" name="verticalLayout_22">
                    <item>
                      <widget class="QLabel" name="label_27">
                        <property name="text">
                          <string>Right:</string>
                        </property>
                      </widget>
                    </item>
                    <item>
                      <widget class="QPushButton" name="btnStickRight">
                        <property name="text">
                          <string />
                        </property>
                      </widget>
                    </item>
                  </layout>
                </item>
                <item row="1" column="0">
                  <layout class="QVBoxLayout" name="verticalLayout_23">
                    <item>
                      <widget class="QLabel" name="label_28">
                        <property name="text">
                          <string>Up:</string>
                        </property>
                      </widget>
                    </item>
                    <item>
                      <widget class="QPushButton" name="btnStickUp">
                        <property name="text">
                          <string />
                        </property>
                      </widget>
                    </item>
                  </layout>
                </item>
                <item row="1" column="1">
                  <layout class="QVBoxLayout" name="verticalLayout_24">
                    <item>
                      <widget class="QLabel" name="label_26">
                        <property name="text">
                          <string>Down:</string>
                        </property>
                      </widget>
                    </item>
                    <item>
                      <widget class="QPushButton" name="btnStickDown">
                        <property name="text">
                          <string />
                        </property>
                      </widget>
                    </item>
                  </layout>
                </item>
              </layout>
            </widget>
          </item>
          <item row="2" column="1">
            <widget class="QGroupBox" name="faceButtons_6">
              <property name="title">
                <string>System Buttons</string>
              </property>
              <property name="flat">
                <bool>false</bool>
              </property>
              <property name="checkable">
                <bool>false</bool>
              </property>
              <layout class="QGridLayout" name="gridLayout_6">
                <item row="0" column="0">
                  <layout class="QVBoxLayout" name="verticalLayout_25">
                    <item>
                      <widget class="QLabel" name="label_29">
                        <property name="text">
                          <string>Start:</string>
                        </property>
                      </widget>
                    </item>
                    <item>
                      <widget class="QPushButton" name="btnStart">
                        <property name="text">
                          <string />
                        </property>
                      </widget>
                    </item>
                  </layout>
                </item>
                <item row="0" column="1">
                  <layout class="QVBoxLayout" name="verticalLayout_26">
                    <item>
                      <widget class="QLabel" name="label_30">
                        <property name="text">
                          <string>Select:</string>
                        </property>
                      </widget>
                    </item>
                    <item>
                      <widget class="QPushButton" name="btnSelect">
                        <property name="text">
                          <string />
                        </property>
                      </widget>
                    </item>
                  </layout>
                </item>
                <item row="1" column="0">
                  <layout class="QVBoxLayout" name="verticalLayout_27">
                    <item>
                      <widget class="QLabel" name="label_31">
                        <property name="text">
                          <string>Home:</string>
                        </property>
                      </widget>
                    </item>
                    <item>
                      <widget class="QPushButton" name="btnHome">
                        <property name="text">
                          <string />
                        </property>
                      </widget>
                    </item>
                  </layout>
                </item>
              </layout>
            </widget>
          </item>
        </layout>
      </item>
      <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
            <spacer name="horizontalSpacer">
              <property name="orientation">
                <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
                <size>
                  <width>40</width>
                  <height>20</height>
                </size>
              </property>
            </spacer>
          </item>
          <item>
            <widget class="QPushButton" name="btnRestoreDefaults">
              <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                </sizepolicy>
              </property>
              <property name="sizeIncrement">
                <size>
                  <width>0</width>
                  <height>0</height>
                </size>
              </property>
              <property name="baseSize">
                <size>
                  <width>0</width>
                  <height>0</height>
                </size>
              </property>
              <property name="layoutDirection">
                <enum>Qt::LeftToRight</enum>
              </property>
              <property name="text">
                <string>Restore Defaults</string>
              </property>
            </widget>
          </item>
        </layout>
      </item>
    </layout>
  </widget>
  <resources />
  <connections />
</ui>